PWE.html

```diff
@@ -30,61 +32,118 @@
+<ul><li>get help and support to safely resolve issues that negatively affect them</li>
+<li>correct their behaviors that negatively affect other participants in W3C</li>
+        <li>continue to safely work and contribute to the W3C after an issue has been raised</li>
+</ul>
+<p>The Code of Ethics and Professional Conduct is the standard that all W3C participants are required to follow and is used as the basis for any resolution and disciplinary actions. In exceptional circumstances, individuals may have their W3C participation suspended or withdrawn as a result of a complaint or investigation.</p>
+
+<p>All situations are different and W3C participants are encouraged to use whatever method of dispute resolution they feel comfortable with.
+</p>
+<p>Individuals are encouraged to try and resolve issues themselves in the first instance. However, if individuals do not feel comfortable or safe doing so, they can get help through the ombudsperson, facilitators and/or mediation. 
+</p>
+<p>Depending on the severity of the situation, it may not be appropriate to attempt to resolve the issue informally (for example, threats of violence, assault or sexual assault). Individuals should immediately report the incident whenever they feel unsafe or threatened by the behaviour or actions of other W3C participants.
+</p>
+<p>Everyone involved in the dispute resolution process is expected to:</p>
+<ul>
+     <li> show respect for others</li>
+    <li> work together to resolve the complaint</li>
+<li> maintain confidentiality</li>
+</ul>
+
+        <p>A disciplinary investigation can be started where one of the following apply:</p>
+<ul><li>it’s not appropriate to begin with mediation </li>
+<li> all other mediation or resolution attempts have not worked</li>
+</ul>
+
+         <section id="procedures">
+            <h2>Incident Resolution Procedures</h2>
+
+            <p>When an incident arises it is important for everyone involved to know what will happen and what resources are available to them. Any time a Code of Conduct violation or complaint occurs it is important that those involved, especially the party/parties affected by the incident, feel supported and respected.</p>
+
+            <p>How an incident is handled will depend on a number of factors including the type of incident, severity, frequency, and contributing factors (for example, has anyone been involved in incidents before, is there a pattern?).</p>
+
+            <p>The procedure for addressing an incident will include some or all of the following steps:</p>
+
+            <ol>
+                <li>Incident is addressed directly with the parties involved, if appropriate.</li>
+                <li>Incident is reported.
+                    <ul>
+                        <li>Reporting can take on many forms including reporting the issue to a chair, ombudsperson, through an anonymous report, or a member of the team.</li>

```

Comment:
I think we should actively encourage people to report incidents, and explain why it is important to do so, even if they think they can resolve them without help.  Reason: It enables patterns and repeat offenders to be identified.